src/main/webapp/app/entities/grade-step.model.ts

```diff
@@ -0,0 +1,9 @@
+export class GradeStep {
```

Comment:
please follow the implementation of other entity classes:
* implement BaseEntity
* make all attributes public
* if you cannot guarantee that the attributes are set, then please use the ? operator to make them optional